### PR TITLE
implemented people table with sort and filter

### PR DESCRIPTION
**Pull request by tiserett**
[DEMO LINK](https://tiserett.github.io/react_dynamic-list-of-todos/)

### CODE REVIEW

File: src/components/TodoModal/TodoModal.tsx [hunk 1]
```diff
@@ -1,12 +1,41 @@
...
+    return null;
+  }
 
-export const TodoModal: React.FC = () => {
   return (
     <div className="modal is-active" data-cy="modal">
       <div className="modal-background" />
 
-      {true ? (
+      {!todoId ? (
```
Comment (Githi54) [inline src/components/TodoModal/TodoModal.tsx:38]:
```suggestion
      {!user ? (
```

File: src/components/TodoModal/TodoModal.tsx [hunk 2]
```diff
@@ -15,31 +44,44 @@ export const TodoModal: React.FC = () => {
...
+              >
+                {currentTodo?.completed ? 'Done' : 'Planned'}
+              </strong>
 
               {' by '}
 
-              <a href="mailto:Sincere@april.biz">
-                Leanne Graham
-              </a>
+              {!user ? (
```
Comment (Githi54) [inline src/components/TodoModal/TodoModal.tsx:78]: you don't need this check

**Review by Githi54 — CHANGES_REQUESTED** [review]
Great start!😁 Please, make some fixes

**Review by Githi54 — APPROVED** [review]
well done!🚀

**Review by danylo-onishchuk — APPROVED** [review]

**Review by danylo-onishchuk — APPROVED** [review]

**Pull request closed without merging**